requirements-dev.txt

```diff
@@ -36,4 +36,4 @@ openpyxl
 # by leveraging openpyxl, see also
 # https://stackoverflow.com/questions/65254535/xlrd-biffh-xlrderror-excel-xlsx-file-not-supported
 # We can remove this upperbound when our minimum pandas version is 0.25+.
-xlrd<2.0.0
+xlrd<2.0.0
```

Comment:
Seems like there is no difference here? I guess a line break was removed? Could you revert this?

Reply:
Fixed